corehq/blobs/models.py

```diff
@@ -4,3 +4,17 @@
 class BlobMigrationState(models.Model):
     slug = models.CharField(max_length=20, unique=True)
     timestamp = models.DateTimeField(auto_now=True)
+
+
+class BlobExpiration(models.Model):
+    '''
+    This models records when temporary blobs should be deleted
+    '''
+    bucket = models.CharField(max_length=255)
+    identifier = models.CharField(max_length=255)
+    expires_on = models.DateTimeField(db_index=True)
+    created_on = models.DateTimeField(auto_now=True)
+    length = models.IntegerField()
+
+    # This is set to True when the blob associated has been successfully deleted
+    deleted = models.BooleanField(default=False, db_index=True)
```

Comment:
https://github.com/dimagi/commcare-hq/pull/15610/commits/9926d613222b87a522e037906e3e0c7fa45e3d8a the fsdb blocks blobs from being updated if the path already exists. so i doubt we'll ever have to worry about this at this level. however i made the change just to future proof it in case we ever want to allow blobs to be updated and because it wasn't that difficult to do